app/controllers/messages_controller.rb

```diff
@@ -21,4 +22,15 @@ def create
   def message_params
     params.require(:message).permit(:content)
   end
+
+  def notify_recipient
+    users_in_room = @message.joined_users
```

Comment:
takes users of chatroom, and sends notification only to recipient (not sender of course)